scheduler/src/migrations/20240926122714_create_jobs_table.sql

```diff
@@ -0,0 +1,31 @@
+-- Create the jobs table
+CREATE TABLE IF NOT EXISTS jobs (
+  id UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+  url VARCHAR(255),
+  routing_key VARCHAR(255),
+  details JSONB,
+  created_at TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP,
+  updated_at TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP
+);
+
+-- Delete bms_data where job_id does not exist in jobs table (foreign key constraint)
+DELETE FROM bms_data WHERE job_id NOT IN (SELECT id FROM jobs);
```

Comment:
@kacperzuk-neti I've assumed that at this point we dont need old data. the format will probably change anyways in near future, with multiple/batch bandwidth tests